src/app/recover-funds/[address]/RecoverFundsButton.tsx

```diff
@@ -133,7 +133,13 @@ function RecoverFundsButton({
     } finally {
       setLoading(false);
     }
-  }, [balanceToRecover, chainID, destinationAddress, signer]);
+  }, [
+    addressToRecoverFrom,
```

Comment:
Why is this dep array changed?

Reply:
It's needed on those lines https://github.com/OffchainLabs/retryable-tx-panel/pull/58/files/ce73a0fcb8ba1206668e826184db79a192514f19#diff-cc55841037b8f7e0687ab0cb53e6301f3650f8d955300d6a0718e171ae5422ebR38-R40 but it was missing before